scripts/gen_vid_diffusion.py

```diff
@@ -346,7 +346,8 @@ def generate(
                 bbox_select[3] = min(img.shape[0], bbox_select[3])
             else:
                 bbox = bboxes[bbox_idx]
-
+            opt.data_online_creation_load_size_A = (1280, 720)
```

Comment:
In general we don´t want hardcoded values here.

Reply:
we temporarily did this to do inference with your model of ` bdd100k_vid_64_2`, since in this model opt.data_online_creation_load_size_A is 720.  Normally, this hardcoded line is not required.